### ISSUE TITLE
Issue when attempting to connect to a twitch stream

### ISSUE THREAD

**Issue by collinsurvive** [streamlink/streamlink#1908]
## Bug Report

- [X ] I have read the contribution guidelines. 

### Description

Version: Most Recent
Player: VLC, most recent
Type of Stream: Twitch.tv (occurred on various larger names such as Lirik/shroud/moonmoon_ow along with smaller names such as Zizaran)
Issue: The player anywhere from 30s to 1min in stops and returns this: https://i.imgur.com/MXY520d.png
Oddly enough it was working fine this morning (around 5-6 hours ago) and just began having this issue 

### Expected / Actual behavior
I expected the stream to play normally like it does in a browser set up, it decides to shut off at a specific interval though.

### Reproduction steps / Explicit stream URLs to test

Happens everytime I try and load a stream through streamlink, nothing has changed in my steps.

### Logs

![image](https://user-images.githubusercontent.com/40900602/42397821-03126556-812d-11e8-995a-e6ce3c9ab1d1.png)

[Love Streamlink? Please consider supporting our collective. Thanks!](https://opencollective.com/streamlink/donate)

**Comment by Githubaccountlol** [comment]
Same for me, new behavior as of today.  Same behavior in the log.

**Comment by gravyboat** [comment]
I'm not able to duplicate this. I just watched a large stream for 10 minutes without issue.

**Comment by collinsurvive** [comment]
> I'm not able to duplicate this. I just watched a large stream for 10 minutes without issue. Are you still encountering this?

Yeah, just checked shroud's stream again and I'm in the same reloading playlist loop. Its a bit odd because it works fine in browser and worked just fine earlier this morning and now it just doesn't work without any changes made to my system nor network.

![image](https://user-images.githubusercontent.com/40900602/42403724-cb29922e-8148-11e8-99fb-93a6945419ca.png)

The reloading playlist continues for a while (30+ lines) until it gives:
![image](https://user-images.githubusercontent.com/40900602/42403735-e8c9bcaa-8148-11e8-8570-d9a38188d4df.png)


If there is any more information I can provide please let me know, otherwise I'm just generally confused.

**Comment by bastimeyer** [comment]
I can't reproduce this either.

@collinsurvive You don't seem to be using the latest version, because it doesn't show the debug information at the beginning of the output, which was added a couple of versions back. Log output should also be posted in text format and not via screenshots, because this kind of information can't be indexed by github or search engines and also can't be copied.

> error: Error when reading from stream: Read timeout, exiting

This might be an issue on Twitch's side, probably an issue in their network. Where are you accessing the stream from?

**Comment by collinsurvive** [comment]
Sorry, I'm using the latest stable, 0.14.2.

I'll be sure to export to a text document if the issue persists or returns.

I think it may have been on twitch side as of 10 mins ago I've been able to have it work again.  Otherwise, I really have no clue to be honest.  If a moderator wants to mark this as closed feel free.  Again, I'm not sure what caused the issue at all, but it seems to have resolved its self.

**Comment by gravyboat** [comment]
I'll close it for now and if it happens again we can always reopen it.

**Comment by back-to** [comment]
@collinsurvive 

> I'm using the latest stable, 0.14.2

_for your interest_

your **log** output does **not** look like the **latest version**,
it look like `0.9.0`

use `streamlink --version` to find out

for more informations see

https://github.com/streamlink/streamlink/issues/1391#issuecomment-354275115
https://github.com/streamlink/streamlink/issues/1340

**Comment by collinsurvive** [comment]
@back-to 

Thanks for the comment links, the streamlink command line was still referencing the 0.10.0 version.  Took care of that, cheers.

**Comment by Githubaccountlol** [comment]
I ran the latest installer when this happened to me, I don't know what it did, but it certainly didn't change what was in the directory I pointed it to.  Didn't occur to me to check the version again afterwards, but that revealed the problem.  Derp.

FWIW, and this is user error, to be sure - the installer doesn't catch your stupidity if you've been using a portable install and try to run it on top of that.  It's been long enough since I needed to update (that's praise, I guess!) that I screwed it up.  Thanks guys.
